ietf/doc/expire.py

```diff
@@ -86,10 +87,15 @@ def send_expire_warning_for_draft(doc):
         (doc.get_state_slug("draft") != "active")):
         return # don't warn about dead or inactive documents
 
-    expiration = doc.expires.astimezone(DEADLINE_TZINFO).date()
+    expiration = doc.expires.astimezone(
+        DEADLINE_TZINFO
+    ).replace(
+        hour=0, minute=0, second=0, microsecond=0
+    ).astimezone(
```

Comment:
Don't actually need to convert the timezone to UTC - the `|utc` in the template will do the right thing (also, just as a non-obvious fyi, we're trying to use `datetime.timezone.utc` as the UTC zone rather than `ZoneInfo("UTC")` or any of the various other synonyms)